Ofqual.Recognition.Frontend.Web/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,76 @@
+﻿﻿@{
+    var controllerName = this.ViewContext.RouteData.Values["controller"]?.ToString();
+}
+
+<!DOCTYPE html>
+<html lang="en" class="govuk-template">
+
+<head>
+    <meta charset="utf-8">
+    <title>@ViewData["Title"] - Ofqual - GOV.UK</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+    <meta name="theme-color" content="#0b0c0c">
+    <link rel="icon" sizes="48x48" href="~/assets/images/favicon.ico">
+    <link rel="icon" sizes="any" href="~/assets/images/favicon.svg" type="image/svg+xml">
+    <link rel="mask-icon" href="~/assets/images/govuk-icon-mask.svg" color="#0b0c0c">
+    <link rel="apple-touch-icon" href="~/assets/images/govuk-icon-180.png">
+    <link rel="manifest" href="~/assets/manifest.json">
+    <link rel="stylesheet" href="~/css/govuk-frontend.min.css" asp-append-version="true">
+    <link rel="stylesheet" href="~/css/application.css" asp-append-version="true" />
+    <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
+    <script type="text/javascript" src="~/js/analytics.js" asp-append-version="true"></script>
```

Comment:
This is currently trying to import a script that doesn't yet exist - I presume the page is still working though? If so we can leave it in for now seeing as we're still prerelease